src/index.html

```diff
@@ -8,9 +8,51 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <div class="container"> -->
+    <form class="searchBar"
+          action="https://www.google.com.ua/search"
+          method="get"
+          data-qa="big"
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 5 - fix formatting according to recommendations (GOOD EXAMPLE)

fix it everywhere